frontend/src/container/BillingContainer/BillingContainer.tsx

```diff
@@ -371,6 +384,17 @@ export default function BillingContainer(): JSX.Element {
 		</Typography>
 	);
 
+	const handleCsvDownload = useCallback((): void => {
+		const csv = unparse(csvData);
+		const csvBlob = new Blob([csv], { type: 'text/csv;charset=utf-8;' });
+		const csvUrl = URL.createObjectURL(csvBlob);
+		const downloadLink = document.createElement('a');
+		downloadLink.href = csvUrl;
+		downloadLink.download = `test-csv-download.csv`;
```

Comment:
fileName is test-csv-download ? change this to billing usage or something based on date range might also work

Reply:
done, new format is - `billing_usage_(06_02_2024-06_06_2024)`